src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
I think making stuff non-public can be overdone, but I am not sure it is the case here. The method is internal (and static). The solution to the problem would be to make the method public, though I honestly am not sure if it is worth it given it's six simple lines of code. There are many other non-public methods that could theoretically be reused, and I hope we won't be making all of them public.